### PR TITLE
Support for multiple continuous queries

### PR DESCRIPTION
**Pull request by vjuranek**
Covers also PR #290

### CODE REVIEW

File: plugins/infinispan80/src/main/resources/jgroups-udp.xml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<config xmlns="urn:org:jgroups"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="urn:org:jgroups file:schema/JGroups-2.8.xsd
```
Comment (mcimbora) [inline plugins/infinispan80/src/main/resources/jgroups-udp.xml]: Looks like a quite an old config, e.g. MERGE2 has been deprecated. Why not using default-jgroups-udp.xml that ISPN ships with?

**Comment by vjuranek** [comment]
Good idea @mcimbora , I updated PR accordingly

File: plugins/infinispan80/src/main/resources/query.xml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  ~ 02110-1301 USA
+  -->
+
+<infinispan
+   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="urn:infinispan:config:8.0 http://www.infinispan.org/schemas/infinispan-config-8.0.xsd"
+   xmlns="urn:infinispan:config:8.0">
+
+   <jgroups>
+      <stack-file name="jgroupsStack" path="${infinispan.jgroups.config:default-jgroups-udp.xml}"/>
```
Comment (mcimbora) [inline plugins/infinispan80/src/main/resources/query.xml]: default-configs/default-jgroups-udp.xml is the correct path to default config, unfortunately this one doesn't work.

**Comment by vjuranek** [comment]
err, sorry, fixed now (I did the change in RG target dir only when testing it :-( )

**Comment by mcimbora** [comment]
Integrated, thanks.

**Pull request closed without merging**